### ISSUE TITLE
Timeline goes onto 2nd row

### ISSUE THREAD

**Issue by richard-still** [namespace-ee/react-calendar-timeline#476]
Timeline goes onto 2nd row, 

I think there are sometimes issues with the way the width of the viewport is calculated, which in some circumstances can make the timeline go onto two rows like the screenshot below. Resizing the screen forces a recalculation and it renders properly. You can also see this in the codesandbox examples if you slowly resize the preview pane to be smaller, the items jump down (to be in the same position as this screenshot) then they re-render to be in the right place.

![image](https://user-images.githubusercontent.com/34010145/49475167-6a36b180-f80e-11e8-97b2-eab12562a259.png)

Related PR - #472

**Comment by Ilaiwi** [comment]
yep I reproduced it in https://codesandbox.io/s/ypy86zww9v . Thanks for noticing that and for the fix. I will make sure the example works after merging

**Comment by Ilaiwi** [comment]
Ok, I spent some time trying to reproduce this. So to reproduce the problem visit this URL https://ypy86zww9v.codesandbox.io/
to reproduce
1- hover over the left edge of the gray div till resize pointer appear
2- resize the timeline by dragging the edge right and left
So as you can see the problem is not with the calendar div that you added `flex` to but is with the sidebar header collapsing. 
This is an issue I took into mind in # 349. 
I have the worked from that branch deployed in its own package for testing purposes. and the problem is solved
you can check it out here https://zlvp6nzx1x.codesandbox.io/
Please if you have time consider helping us finish #391. We need help in unit testing the implementation.
